vtm.js

```diff
@@ -245,7 +245,7 @@ const app = Vue.createApp({
       },
       disciplines:{
         id: "Disciplines",
-        resource: [0,1,1,0,0,0],
+        resource: [8,1,1,0,0,0],
```

Comment:
jop toto bolo spatne... je to ale zamerne spatne lebo je potreba aby sa to nastavovalo na 10,0,0 alebo 8,1,1 podla blood potency